### PR TITLE
Refactor ListOfGames component

### PR DESCRIPTION
**Pull request by DRFR0ST**
It also fixes the date typo.

### CODE REVIEW

File: src/components/Home/ListOfGames/index.tsx [hunk 1]
```diff
@@ -10,42 +9,91 @@ import {
...
+      <>
+        <Title>{props.name}</Title>
+
+        <Paragraph>{props.first_release_date}</Paragraph>
+        <Paragraph>{summary}</Paragraph>
+      </>
+      <Score>{props.rate}</Score>
+    </TextWrapper>
+  </GameArticle>);
+}
```
Comment (DRFR0ST) [inline src/components/Home/ListOfGames/index.tsx:43]: You should probably move this component to a separate file.

File: src/components/Home/ListOfGames/index.tsx [hunk 2]
```diff
@@ -10,42 +9,91 @@ import {
...
+  /** Triggers the fetch once on initialization. You can propagate the fetchGames function from this hook if you need to refetch. */
+  useEffect(() => {
+    fetchGames();
+  }, []);
+
+  return games as TGame[];
+}
+
+/** Handles the whole parsing process for a single game entry. */
+const parseGame = (raw_game: TRawGame): TGame => ({ ...raw_game, first_release_date: new Date(raw_game.first_relase_date), rate: Math.floor(raw_game.rating / 10) });
```
Comment (DRFR0ST) [inline src/components/Home/ListOfGames/index.tsx:97]: This hook should also be placed in a separate file. Same goes for the types, a simple `types.ts` file will be enough to store these two bad bwoys. 🤠

File: src/components/Home/ListOfGames/index.tsx [hunk 3]
```diff
@@ -10,42 +9,91 @@ import {
...
+  summary: string;
+}
+
+type TGame = {
+  id: number;
+  name: string;
+  rate: number;
+  summary: string;
+  first_release_date: Date;
+}
```
Comment (DRFR0ST) [inline src/components/Home/ListOfGames/index.tsx:62]: That's some rather **meh** repetition, you can play around with the types if you like to make it less repetitive.

File: src/components/Home/ListOfGames/index.tsx [hunk 4]
```diff
@@ -10,42 +9,91 @@ import {
...
+const URL = "https://public.connectnow.org.uk/applicant-test/";
 
+const ListOfGames = () => {
+  const gamesList = useGamesList();
 
   return (
     <DataWraper>
-      {data.map((game) => {
-        const rate: number = Math.floor(game?.rating / 10);
-        const date: any = game?.first_relase_date
```
Comment (DRFR0ST) [inline src/components/Home/ListOfGames/index.tsx:30]: Here is the typo that I've mentioned 🐛

File: src/components/Home/ListOfGames/index.tsx [hunk 5]
```diff
@@ -10,42 +9,91 @@ import {
   Title,
 } from "./styles";
 
-const ListOfGames = () => {
-  const url = "https://public.connectnow.org.uk/applicant-test/";
```
Comment (DRFR0ST) [inline src/components/Home/ListOfGames/index.tsx:14]: No need to store it inside the component. At best keep it outside a component's lifecycle if you can.

File: src/components/Home/ListOfGames/index.tsx (hunk 2 above)
Comment (cieslikprzemyslaw) [reply src/components/Home/ListOfGames/index.tsx:97]: Cheers for drop eye on repo and add some stuff :)

**Pull request merged**